### PR TITLE
LPS-133406 | master

### PR DESCRIPTION
**Pull request by ThalitaCeli**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase [hunk 1]
```diff
@@ -388,6 +388,36 @@ definition {
 		AssertHTMLSourceTextPresent(value1 = "${escapedScript}");
 	}
 
+	@description = "This is for LPS-117983."
+	@priority = "4"
+	test ViewStructureName {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase:393]: AddWebContentBasedOnStructureNameWithXSS

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase [hunk 2]
```diff
@@ -388,6 +388,36 @@ definition {
...
+		task ("Create a structure named <script>alert(123)</script>") {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase:397]: Navigate to Web Content admin via URL

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase [hunk 3]
```diff
@@ -388,6 +388,36 @@ definition {
...
+		}
+
+		task ("Save the Structure") {
+			WebContentStructures.saveCP();
+		}
+
+		task ("View the Structure name") {
+			WebContentStructures.viewCP(structureName = '''<script>alert(123);</script>''');
+		}
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase:419]: Add step to add web content based on new structure

**Review by Tim-Cao — COMMENTED** [review]
Hi @ThalitaCeli, Could you move the case to portalweb\tests\enduser\wem\journal\WebContentWithXSS.testcase ? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase (hunk 3 above)
Comment (ThalitaCeli) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase]: Hello @Tim-Cao,
I couldn't Add Web Content based on new Structure called `<script>alert(123);</script>`. The structure appears to be added as  `&lt;script&gt;alert(123);&lt;/script&gt;`

Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase]: No need to type name. The case test no alert popup when create a web content based on structure with XSS name.

**Comment by Tim-Cao** [comment]
Hi @ThalitaCeli, see changes https://github.com/liferay-echo/liferay-portal/compare/35eed29605f8108ebd1807a2d87410995c7e095f...489c382af3a1b2667975d5fc04495835568af777

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**